Tests iOS/Fixtures/archived-web-view.json

```diff
@@ -17,7 +17,7 @@
             "node": {
               "__typename": "[type-name-here]",
               "remoteID": "archived-item-1",
-              "url": "http://localhost:8080/hello",
+              "url": "http://localhost:8080/hello1",
```

Comment:
can we have more descriptive names for URLs?

Reply:
yeah, a lot of them are unfinished with unclear names since I wasn't feeling well wed night and just pushed up a draft PR EOD. will update!